pkg/kubelet/userns/types.go

```diff
@@ -0,0 +1,13 @@
+package userns
```

Comment:
This is missing the copyright header, which is failing CI:
https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/130800/pull-kubernetes-verify/1900264114270244864